litho-it/src/test/java/com/facebook/litho/widget/RecyclerEventsControllerTest.java

```diff
@@ -72,52 +86,61 @@ public void testOnRecyclerListener() {
 
   @Test
   public void testClearRefreshingOnNotRefreshingView() {
-    when(mSectionsRecyclerView.isRefreshing()).thenReturn(false);
-
+    setUpInitialValues(false);
     mRecyclerEventsController.clearRefreshing();
 
-    verify(mSectionsRecyclerView, never()).setRefreshing(anyBoolean());
-    verify(mSectionsRecyclerView, never()).removeCallbacks(any(Runnable.class));
-    verify(mSectionsRecyclerView, never()).post(any(Runnable.class));
+    assertThat(mSectionsRecyclerView.getSetRefreshingValues()).isEmpty();
```

Comment:
Let's not leave this one alone and extract it to `verifySetRefreshingWasCalledNTimes` too

Reply:
Done